### PR TITLE
samples/jsfiddle-2-samples-4

### PR DESCRIPTION
**Pull request by sigrunnu**
Found a few more demos that needs to be moved

### CODE REVIEW

**Comment by TorsteinHonsi** [comment]
While you're at it, can you remove this one `highcharts/blog/activity-gauge` ? And in the blog, we would include `highcharts/demo/gauge-activity` instead. The first one seems to be a copy of the demo, except it doesn't work well.

**Comment by sigrunnu** [comment]
I'll remove it! Also, do you know why the review tool is failing?
@TorsteinHonsi

File: samples/highcharts/blog/aapl-price/demo.details [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+ name: AAPL Price
```
Comment (TorsteinHonsi) [inline samples/highcharts/blog/aapl-price/demo.details:2]: It's the same as http://localhost:3030/samples/#view/stock/demo/basic-line, use that instead

File: samples/highcharts/blog/color-blindness/demo.details [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+ name: How people with color blindness see colors
+ authors: Mustapha Mekhatria
+   -
```
Comment (TorsteinHonsi) [inline samples/highcharts/blog/color-blindness/demo.details:4]: Funny dash - probably not valid YAML

File: samples/highcharts/blog/movie-cummulative-income/demo.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+
+<div id="container" style="margin: 0; height:600px"></div>
```
Comment (TorsteinHonsi) [inline samples/highcharts/blog/movie-cummulative-income/demo.html:5]: Move the inline CSS to demo.css

File: test/json-sources.js [hunk 4]
```diff
@@ -8,6 +8,12341 @@
  * and used from karma-setup.js.
  */
 window.JSONSources = {
+    "https://cdn.jsdelivr.net/gh/highcharts/highcharts@v7.0.0/samples/data/usdeur.json":
```
Comment (TorsteinHonsi) [inline test/json-sources.js:11]: This is probably not needed as we remove the stock demo and use our existing stock line

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! Some minor adjustments.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**